addons/website_slides/views/website_slides_templates_course.xml

```diff
@@ -561,24 +560,25 @@
 
                 <a t-if="slide_promoted.is_preview or channel.is_member or is_slides_publisher"
                 t-attf-href="/slides/slide/#{slug(slide_promoted)}#{query_string}" class="w-100 w-lg-50 flex-shrink-0 rounded">
-                    <div t-attf-style="background-image:url(#{image_url}); background-size: cover; background-position:center; padding-bottom: 35%;" class="h-lg-100"/>
+                    <div t-if="slide_promoted.image_1920" t-field="slide_promoted.image_1920" t-options="{'widget': 'image', 'style': 'height:100%', 'preview_image': 'image_1024'}" class="h-100"/>
```

Comment:
Not sure what "preview_image" does, but from what I see in the current usage it's usually a smaller version (like _128 or _254), maybe to double-check?